### ISSUE TITLE
Please update documentation for Update user

### ISSUE THREAD

**Issue by asorokhtey** [strimzi/strimzi-kafka-operator#770]
We can move this part separately 
```
Or edit the KafkaUser resource using OpenShift or Kubernetes tools. On Kubernetes this can be done using kubectl edit:

kubectl edit kafkauser <your-user-name>
On OpenShift this can be done using oc edit:
oc edit kafkauser <your-user-name>
```

Basically, you have to prepare YAML file and then run command
`oc apply -f  user.yaml`
Or run command
`oc edit kafkauser my-user` and then in the command line edit yaml and save changes

Now in documentation looks like we have to execute both steps.

**Comment by dmuntima** [comment]
@asorokhtey, I have raised https://github.com/strimzi/strimzi-kafka-operator/pull/780 PR as a fix for this issue.

**Comment by dmuntima** [comment]
This issue is now fixed (PR #780).
